Userland/Services/WindowServer/WindowServer.ipc

```diff
@@ -5,6 +5,7 @@
 endpoint WindowServer
 {
     create_menu(i32 menu_id, [UTF8] DeprecatedString menu_title) =|
+    set_menu_title(i32 menu_id, DeprecatedString menu_title) =|
```

Comment:
Awkward that we call these "name" inside WindowServer but "title" in the IPC API. :thinking:

Reply:
I did that to be consistent with  `create_menu()`, but since that's the only place it's called "menu_title" I'll just rename that to be consistent with everything else :)